iina/zh-Hans.lproj/QuickSettingViewController.strings

```diff
@@ -165,7 +165,7 @@
 "bzk-c2-LH5.label" = "音频";
 
 /* Class = "NSSegmentedCell"; cMu-YF-riv.ibShadowedLabels[0] = "Default"; ObjectID = "cMu-YF-riv"; */
-"cMu-YF-riv.ibShadowedLabels[0]" = "Default";
+"cMu-YF-riv.ibShadowedLabels[0]" = "默认";
```

Comment:
I see,i've tried to change them.
But I think the better way to select a menu item is using id,not the title of that item because the title may be translated.
So,why don't we use English for coding and localized string for displaying?